### ISSUE TITLE
simulate() not properly working for ordbeta-family?

### ISSUE THREAD

**Issue by strengejacke** [glmmTMB/glmmTMB#942]
`simulate()` always returns identical results, it seems. Does this currently not work for the `ordbeta` family?

``` r
library(glmmTMB)
library(lme4)
#> Loading required package: Matrix
library(datawizard)

data(sleepstudy)
sleepstudy$y <- normalize(sleepstudy$Reaction)

m <- glmmTMB(y ~ Days + (1 | Subject), data = sleepstudy, family = ordbeta)

head(simulate(m, nsim = 5))
#>       sim_1     sim_2     sim_3     sim_4     sim_5
#> 1 0.2030275 0.2030275 0.2030275 0.2030275 0.2030275
#> 2 0.2366451 0.2366451 0.2366451 0.2366451 0.2366451
#> 3 0.2075882 0.2075882 0.2075882 0.2075882 0.2075882
#> 4 0.4672708 0.4672708 0.4672708 0.4672708 0.4672708
#> 5 0.5974521 0.5974521 0.5974521 0.5974521 0.5974521
#> 6 0.8100759 0.8100759 0.8100759 0.8100759 0.8100759

head(simulate(m, nsim = 1))
#>       sim_1
#> 1 0.2030275
#> 2 0.2366451
#> 3 0.2075882
#> 4 0.4672708
#> 5 0.5974521
#> 6 0.8100759
head(simulate(m, nsim = 1))
#>       sim_1
#> 1 0.2030275
#> 2 0.2366451
#> 3 0.2075882
#> 4 0.4672708
#> 5 0.5974521
#> 6 0.8100759
head(simulate(m, nsim = 1))
#>       sim_1
#> 1 0.2030275
#> 2 0.2366451
#> 3 0.2075882
#> 4 0.4672708
#> 5 0.5974521
#> 6 0.8100759
```

<sup>Created on 2023-09-22 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by bbolker** [comment]
Oversight. Shouldn't be too hard to implement, I think/hope ...

**Comment by bbolker** [comment]
Preliminary implementation available [here](https://github.com/glmmTMB/glmmTMB/tree/ordbeta_sim) (`remotes::install_github("glmmTMB/glmmTMB/glmmTMB@ordbeta_sim")`, I think ...)

**Comment by strengejacke** [comment]
Looks good!

``` r
library(easystats)
library(glmmTMB)
library(lme4)
data(sleepstudy)
sleepstudy$y <- normalize(sleepstudy$Reaction)

m <- glmmTMB(y ~ Days + (Days | Subject), data = sleepstudy, family = ordbeta())
out <- check_predictions(m)
plot(out)
```

![](https://i.imgur.com/PcsKvPH.png)

Here's the Bayesian equivalent from `ordbetareg`:

```r
m2 <- ordbetareg(y ~ Days + (Days | Subject), data = sleepstudy, chains = 1, iter = 500)
out2 <- check_predictions(m2)
plot(out2)
```

![plot](https://github.com/glmmTMB/glmmTMB/assets/26301769/c167c96c-d13b-4c48-bc1c-55b1f0cfb9ce)

**Comment by strengejacke** [comment]
If you consider this as relatively stable, maybe this can be included in 1.1.8 already... :-)
